environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 
   # build dependencies
   - versioneer[toml]
-  - cython=3.0.5
+  - cython~=3.0.5
```

Comment:
Any reason for pinning to 3.0 instead of a more general `>= 3.0` range?

Reply:
I meant to put this in the OP but forgot: I brought this up at the dev meeting in December. I am wondering if minor releases should be benchmarked first before we upgrade. The general consensus was that patch versions are fine to automatically take, but there was uncertainty around minor versions. 

No opposition to `>=` if people are comfortable with that.

Reply:
cc @jbrockmendel @mrocklin

Reply:
OK cool. I have no objection to this, as long as someone takes the time to do it (your benchmarks were very helpful with the 3.x upgrade)